src/pages/Trade/components/ChartCard/PairListPanel/PairListPanel.tsx

```diff
@@ -77,7 +77,10 @@ const PairListPanel: FC<PairListPanelProps> = ({
 					});
 				} else {
 					const searchLowered = search.toLowerCase();
-					const searchParts = searchLowered.split(' ');
+
+					// try to match "eth / btc", "eth/btc", etc...
+					const searchParts = searchLowered.replace(/([^a-z0-9]+)/gi, ' ').split(' ');
```

Comment:
The idea here is that I turn all alphanumeric chars to spaces, and then split the string to spaces (I only will care about the first and last result), so things like "eth Blah blah blah btc" would still capture "eth btc"